bedrock/redirects/redirects.py

```diff
@@ -1644,6 +1644,10 @@
     redirect('^performance/leak-brownbag\.html$',
              'http://wiki.mozilla.org/Performance:Leak_Tools'),
     redirect('^privacy-policy(\.html|/.*)?$', '/privacy/websites/'),
+    redirect('^projects/calendar/$', 'https://www.thunderbird.net/calendar/'),
+    redirect('^projects/calendar/holidays/$', 'https://www.thunderbird.net/calendar/holidays/'),
+    # bug 885799, 952429
+    redirect(r'^projects/calendar/holidays\.html$', 'https://www.thunderbird.net/calendar/holidays/'),
```

Comment:
Might also be a good idea to redirect the /media urls for the calendar files as well so that we don't get a bunch of bugs filed for missing ical files. Though we may also not want to continue to being in the business of handling all of that traffic from Lightening. Dunno. Would be something like the following if we wanted to do it:

```python
redirect(r'media/caldata/(?P<filename>.+\.ics)$', 'https://www.thunderbird.net/media/caldata/{filename}'),
```

Reply:
I don't really use Lightning. Does it directly link to these files in some way? If so we should probably change the URLs at the source? @kewisch

Reply:
People directly use these links for calendars. I could add an in-product transition mechanism, but this would need time to propagate. Are there stats on how often these are requested?

Reply:
Unfortunately not, at least not for ICS feeds. We have a such thing for caldav calendars and what I would do in product is add that same check for ics feeds.

Reply:
So should we continue hosting the calendar files for a while so that we don't break people's calendars until product changes are made to support redirection?

Reply:
I'm cool with that. @alexgibson @stephaniehobson is that okay with you? We'd just leave those .ics files where they are for now but remove everything else, and the new location for the holiday calendars page and files will be the new thunderbird site. 

@kewisch I'll see what I can find out about the traffic. I'll file an issue in our infra repo and CC you there.

Reply:
Thanks! In reading that ticket I realized I wasn't quite clear in what Lightning can do, I apologize. The code is very well capable to follow 3xx redirects, but it will not "remember" the redirect and continue to request the original url on the next refresh. 

The product change I would like to do is to allow the user to update the url on a 3xx redirect, so that eventually these calendar file redirects will no longer be necessary.

Reply:
Ah! That's cool. So, what should we do here in your opinion? Keep the files for now or go ahead with the redirects? The redirects won't bother us, but I understand if you'd like for us to complete the traffic estimate before we start redirecting to you.

Reply:
I updated the ticket to try to explain the situation more accurately.

Reply:
Cloudflare caches the .ics files for us so I don't think the traffic requirements are that important. Nice to know, of course, because it will be difficult for us to obtain similar stats since we don't currently have any way to access Cloudflare logs.

Reply:
Are you using cloudflare for thunderbird.net as well? And if so why would you not have access to the logs?

Reply:
Cloudflare doesn't provide log access unless you have an Enterprise account, afaik. That's what their support database says anyway. (https://support.cloudflare.com/hc/en-us/articles/216672448-Enterprise-Log-Share-Logpull-REST-API)

Reply:
Ah. Didn't realize that.

Reply:
Did we ever get the CDN logs? I don't think this should block this PR being merged btw.

Reply:
I filed this but haven't heard anything https://github.com/mozmeao/infra/issues/753